Makefile

```diff
@@ -70,6 +73,9 @@ flow:
 lint-frontend:
 	${DC} run --rm -w /app/frontend webapp ./node_modules/.bin/eslint src/
 
+pytest:
```

Comment:
This helps with running tests between different versions of Python. Is it okay to keep it here or do you want to remove it?  `make test` is a heavyweight task, that's why I've added this target.

Reply:
That's completely fine. It's the equivalent of `make test-frontend`. And I actually like this name better, so I'm think of renaming `test-frontend` to `jest`.